client/src/containers/SettingsTab/index.jsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { Grid, Menu, Header, Divider, Input, Button, Message, Loader } from 'semantic-ui-react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { fetchRepoSettings } from '../../routines/routines';
+import { renameRepo, deleteRepo } from './actions';
+import styles from './styles.module.scss';
+
+const renderDangerField = (header, description, buttonName, clickHandler) => (
+    <Message className={styles.dangerField}>
+        <Grid>
+            <Grid.Column width={12}>
+                <Header as="h6">{header}</Header>
+                <p>{description}</p>
+            </Grid.Column>
+            <Grid.Column width={4}>
+                <Button type="button" className={styles.dangerButton} fluid onClick={clickHandler}>
+                    {buttonName}
+                </Button>
+            </Grid.Column>
+        </Grid>
+    </Message>
+);
+
+class RepoSettings extends React.Component {
+    constructor(props) {
+        super(props);
+
+        const { name } = this.props;
+        this.state = {
+            name
+        };
+    }
+
+    componentDidMount() {
+        const { owner, name } = this.props;
+        this.props.fetchRepoSettings({
+            owner,
+            name
+        });
+    }
+
+  onClickDelete = () => {
```

Comment:
Something happen with code formatting. Please check prettier and linter commands on your machine and if you'll find any issues ask @alexandrtovmach